src/tab-manager/index.js

```diff
@@ -98,6 +98,8 @@ const addTabs = ipcSender => tabsMetadata => {
 
     tab.webContents.executeJavaScript(`window.tabId = '${id}';`);
 
+    // tab.webContents.setAudioMuted(!notificationsEnabled);
+
```

Comment:
Dead code should be removed (I guess this is probably here so I can check what we talked, we'll provide feedback accordingly)

Reply:
I removed everything related to muting the tabs, but I checked it again, and inner sounds are still playing for me.